package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "dehumanize-date",
+  "version": "3.0.1",
```

Comment:
The version is handled by https://rollingverisons.com - it is deliberately omitted from the git repository to avoid confusion, since it would not be automatically updated.

Reply:
No worries! I've also had to use prettier on the code so please use this PR as a reference :) I also added Ukrainian and Italian.

Reply:
Thanks, but that isn't really how open source works. If you'd like to contribute these features, that would be great. If you'd like me to make the changes for you, you'll need to start paying my salary.

Reply:
Well, literally no need to be this aggressive :) I don't want you to do anything, just suggested a way I modified your code. I'm perfectly fine using my own fork.

Yes, **this is** how open source works.